source/material_model/grain_size.cc

```diff
@@ -1345,6 +1396,14 @@ namespace aspect
                  ExcMessage("The parameter 'Use paleowattmeter' has been removed. "
                             "Use the parameter 'Grain size evolution formulation instead'."));
 
+          volume_fraction_phase_one             = Utilities::string_to_double
+                                                  (Utilities::split_string_list(prm.get ("Volume fraction of phase")));
+
+          if (volume_fraction_phase_one.size()>1)
```

Comment:
I think we want this to be a double anyway; but even with a vector, you would still need to check that the volume fraction is between zero and one if it only had one entry.